content/03/03.04/script.ruby.adoc

```diff
@@ -0,0 +1,34 @@
+include::./title.adoc[]
+
+One common example is the problem of plurals. Suppose we want to place Lucy and Sean just 1 metre apart. This step would not match:
+
+[source, gherkin]
+----
+include::../code/ruby/04-allow-for-singular-or-plural-metres/features/hear_shout.feature[lines=3..3]
+----
+
+Because we’ve used the singular `metre` instead of the plural `metres` we don’t have a match. What a pain!
```

Comment:
I'd add a cucumber run here in order to "show, don't tell". Just a quick run, show red, come back to the code.

Reply:
done, thanks|!

Reply:
(in 3a8d7957543991467184410b6e379e9c945f61ed)